.github/copilot-instructions.md

```diff
@@ -24,21 +24,22 @@ Always reference these instructions first and fallback to search or bash command
 - Run unit tests: `dotnet test lewee.sln --configuration Release --no-build --nologo --filter "Category!=Integration"`
   - **TIMING**: Takes ~4 seconds. Set timeout to 60+ seconds.
 - Run integration tests: `dotnet test lewee.sln --configuration Release --no-build --nologo --filter "Category=Integration"`
-  - **TIMING**: Takes 5+ minutes with Docker containers. NEVER CANCEL. Set timeout to 600+ seconds minimum.
-  - **NOTE**: Integration tests use TestContainers and require Docker to be running
+  - **TIMING**: Takes 5+ minutes with containers. NEVER CANCEL. Set timeout to 600+ seconds minimum.
+  - **NOTE**: Integration tests use TestContainers and .NET Aspire for PostgreSQL containers
 - Pack NuGet packages: `dotnet pack lewee.sln --configuration Release --nologo --no-build`
   - **TIMING**: Takes ~2 seconds. Set timeout to 60+ seconds.
 
 ### Sample Application
```

Comment:
Add another section at this level `### Coding Style`.

It should mention that the coding styles are in the `.editorconfig` file and that `dotnet format` should be run.